heft-plugins/heft-sass-plugin/src/SassPlugin.ts

```diff
@@ -71,38 +72,54 @@ export class SassTypingsPlugin implements IHeftPlugin {
   ): Promise<ISassConfiguration> {
     const { buildFolder } = heftConfiguration;
     const sassConfigurationJson: ISassConfigurationJson | undefined =
-      await SassTypingsPlugin._getSassConfigurationLoader().tryLoadConfigurationFileForProjectAsync(
+      await SassPlugin._getSassConfigurationLoader().tryLoadConfigurationFileForProjectAsync(
         logger.terminal,
         buildFolder,
         heftConfiguration.rigConfig
       );
 
+    if (sassConfigurationJson) {
+      if (sassConfigurationJson.srcFolder && !path.isAbsolute(sassConfigurationJson.srcFolder)) {
```

Comment:
Is `path.isAbsolute` cheaper than `path.resolve`? Calling `path.resolve(path1, someAbsolutePath)` returns `someAbsolutePath`.

Reply:
Hmm yeah good point. Probably cheaper to do it all at once.